app/harvesters/hathi_harvester.rb

```diff
@@ -0,0 +1,104 @@
+require 'net/http'
+require 'rubygems/package'
+require 'zlib'
+
+
+class HathiHarvester < Krikri::Harvesters::MarcXMLHarvester
+  BASE_URL = 'http://mirlyn-aleph.lib.umich.edu/mdp/dpla_metadata/'
+
+  def initialize(opts = {})
+    if !(opts[:url] || opts[:dump_file])
+      raise "You must provide either `:url` or `:dump_file`"
+    end
+
+    opts[:uri] = (opts[:url] || opts[:dump_file])
```

Comment:
Yeah, this was a bit we weren't sure about.  We could have it take the URI of the dump file itself, but then someone has to do the manual work of figuring out which dump file should be harvested (as opposed to just clicking a "Harvest!" button in some future interface).

But on the other hand, you can imagine someone *wanting* to target a particular dump file.  So it's tricky :)